pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -21,17 +21,18 @@ import (
 	"os"
 	"path/filepath"
 	"runtime"
+	"strings"
 	"testing"
 	"time"
 
 	gomock "github.com/golang/mock/gomock"
 	cadvisorfs "github.com/google/cadvisor/fs"
 	cadvisorapiv2 "github.com/google/cadvisor/info/v2"
+	"github.com/google/uuid"
```

Comment:
let's continue to use `k8s.io/apimachinery/pkg/util/uuid` as before?

Reply:
Changed as suggested.

Reply:
And also added suffix check